packages/api/src/repository/ResultRequest.ts

```diff
@@ -55,9 +55,16 @@ export class ResultRequestRepository {
   }
 
   async insert (resultRequest: Omit<ResultRequestDbObject, '_id'>) {
-    const response = await this.collection.insertOne(resultRequest)
+    if (this.isValidResultRequest(resultRequest)) {
+      const response = await this.collection.insertOne(resultRequest)
 
-    return this.normalizeId(response.ops[0])
+      return this.normalizeId(response.ops[0])
```

Comment:
We're early-returning a value here but the function is missing a return type signature.

Reply:
Currently we are not using an specific linter for `typescript` but we should do it. I've added missing return types on this but I've also created #52 to check it